app/Domains/Users/Policies/UserPolicy.php

```diff
@@ -71,8 +71,8 @@ public function lock(User $user, User $model): bool
     /**
      * Determine whether the authenticated user can unlock an user account or not.
      *
-     * @param  User $user  The resource entity from the authenticated user.
-     * @param  User $model The resource entity from the given user.
+     * @param User $user The resource entity from the authenticated user.
```

Comment:
Hey,

Can u align the User class with the same start as the return type in the docblock.
Also the start of the description text.